saleor/graphql/translations/schema.py

```diff
@@ -58,7 +58,7 @@ class TranslatableKinds(graphene.Enum):
     PAGE = "Page"
     PRODUCT = "Product"
     SALE = "Sale"
-    SHIPPING_METHOD = "ShippingMethod"
+    SHIPPING_METHOD = "ShippingMethodType"
```

Comment:
This should stay as `SHIPPING_METHOD` or should we rename it asa well?

Reply:
I think that those keys more accurately refer to the model, and not the type. So in my opinion it should stay as `SHIPPING_METHOD`